service/controller/control_plane_drainer.go

```diff
@@ -24,9 +24,11 @@ import (
 	"github.com/giantswarm/aws-operator/service/controller/resource/drainerfinalizer"
 	"github.com/giantswarm/aws-operator/service/controller/resource/drainerinitializer"
 	"github.com/giantswarm/aws-operator/service/internal/asg"
+	"github.com/giantswarm/aws-operator/service/internal/recorder"
 )
 
 type ControlPlaneDrainerConfig struct {
+	Event     recorder.Interface
```

Comment:
This reads wrong to me. What about a bit more consistency? 

```suggestion
	Event     event.Interface
```

Reply:
👍